### PR TITLE
Build more architectures

### PR DESCRIPTION
**Pull request by xhochy**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xhochy** [comment]
cc @0xbe7a

File: recipe/build.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+if [[ ("${target_platform}" == "win-64" && "${build_platform}" == "linux-64") ]]; then
+  # we need to add the generate-import-lib feature since otherwise
+  # maturin will expect libpython DSOs at PYO3_CROSS_LIB_DIR
+  # which we don't have since we are not able to add python as a host dependency
+  cargo add pyo3 \
```
Comment (0xbe7a) [inline recipe/build.sh:9]: This only works if "pyo3" is not an existing dependency, which is not the case with this "Cargo.toml".

**Comment by thewchan** [comment]
@0xbe7a @xhochy I believe it's because we were experience the exact same thing at staged-recipe; win build runs out of memory.

**Comment by xhochy** [comment]
@0xbe7a This packages builds `polars` as part of its dependencies, i.e. it needs even more RAM.

**Comment by thewchan** [comment]
@xhochy I'll wait for this to be done first before I work on the new versions one

**Comment by 0xbe7a** [comment]
I opened https://github.com/conda-forge/staged-recipes/pull/25890

**Pull request merged**